web-assembly/README.md

```diff
@@ -0,0 +1,5 @@
+Web Assembly Sample
+===
+See https://googlechrome.github.io/samples/web-assembly/index.html for a live demo.
```

Comment:
If there are going to more samples, I would recommend doing something like https://googlechrome.github.io/samples/web-bluetooth/ and https://googlechrome.github.io/samples/image-capture/